third-party/cmake/DownloadAndExtractFile.cmake

```diff
@@ -22,7 +22,7 @@ set(SRC_DIR ${PREFIX}/src/${TARGET})
 
 # Check whether the source has been downloaded. If true, skip it.
 # Useful for external downloads like homebrew.
-if(EXISTS "${SRC_DIR}" AND IS_DIRECTORY "${SRC_DIR}")
+if(ALLOW_EXISTING_SRC_DIR AND EXISTS "${SRC_DIR}" AND IS_DIRECTORY "${SRC_DIR}")
```

Comment:
@fwalch Just curious, is this what prevented the travis cache from being updated?